### PR TITLE
Amending Encryption Configuration Section

### LINKED ISSUE #4008: [Rancher2] Clarify EncryptionConfiguration for Rancher Backup Operator
**Request Summary:**

Can we use this `EncryptionConfiguration` as-is for the Rancher Backups Operator? Does it really read `key1` and `key2` values?

**Details:**

https://rancher.com/docs/rancher/v2.6/en/backups/examples/#example-encryptionconfiguration

### PR DESCRIPTION
**Pull request by divya-mohan0209**
Fixes #4008

### CODE REVIEW

File: content/rancher/v2.6/en/backups/examples/_index.md [hunk 1]
```diff
@@ -272,6 +272,13 @@ data:
 
 # Example EncryptionConfiguration
 
+The snippet below demonstrates two different types of secrets and their relevance with respect to Backup and Restore of custom resources.s
```
Comment (brandond) [inline content/rancher/v2.6/en/backups/examples/_index.md:275]:
```suggestion
The snippet below demonstrates two different types of secrets and their relevance with respect to Backup and Restore of custom resources.
```

**Review by brandond — CHANGES_REQUESTED** [review]
LGTM; one typo nit

**Review by brandond — APPROVED** [review]

File: content/rancher/v2.6/en/backups/examples/_index.md [hunk 2]
```diff
@@ -272,6 +272,13 @@ data:
 
 # Example EncryptionConfiguration
 
+The snippet below demonstrates two different types of secrets and their relevance with respect to Backup and Restore of custom resources.
+
+The first example is that of a secret that is used to encrypt the backup files. The backup operator, in this case, will not be able to read the secrets encryption file. It only uses the contents of the secret. 
+
+The second example is that of a Kubernetes secrets encryption config file that is used to encrypt secrets when stored in etcd. It is important to note that while backing up your etcd config ensure you also backup your EncryptionConfiguration. Failing to do so might result in you not being able to use the restored data.
```
Comment (jtravee) [inline content/rancher/v2.6/en/backups/examples/_index.md:279]:
```suggestion
The second example is that of a Kubernetes secrets encryption config file that is used to encrypt secrets when stored in etcd. **While backing up your etcd config, be sure to also backup your EncryptionConfiguration.** Failing to do so might result in you not being able to use the restored data. 
```

Comment (jtravee) [reply content/rancher/v2.6/en/backups/examples/_index.md:279]: Suggestion to add emphasis to this warning.

File: content/rancher/v2.6/en/backups/examples/_index.md [hunk 3]
```diff
@@ -276,7 +276,7 @@ The snippet below demonstrates two different types of secrets and their relevanc
 
 The first example is that of a secret that is used to encrypt the backup files. The backup operator, in this case, will not be able to read the secrets encryption file. It only uses the contents of the secret. 
 
-The second example is that of a Kubernetes secrets encryption config file that is used to encrypt secrets when stored in etcd. It is important to note that while backing up your etcd config ensure you also backup your EncryptionConfiguration. Failing to do so might result in you not being able to use the restored data. 
+The second example is that of a Kubernetes secrets encryption config file that is used to encrypt secrets when stored in etcd. **While backing up your etcd config, be sure to also backup your EncryptionConfiguration.** Failing to do so might result in you not being able to use the restored data.
```
Comment (brandond) [inline content/rancher/v2.6/en/backups/examples/_index.md:279]:
```suggestion
The second example is that of a Kubernetes secrets encryption config file that is used to encrypt secrets when stored in etcd. **When backing up the etcd datastore, be sure to also back up the EncryptionConfiguration.** Failure to do so will result in an inability to use the restored data if secrets encryption was in use at the time the data was backed up.
```

**Comment by divya-mohan0209** [comment]
Since all typos & suggestions have been addressed, merging the PR. Please feel free to reopen if any concerns/if there's a need to revert.

**Pull request merged**